files/default/attachVolume.py

```diff
@@ -34,8 +36,17 @@ def main():
     # List of available block devices after removing currently used block devices
     availableDevices = [a for a in blockDevices if a not in paths]
 
+    # Parse configuration file to read proxy settings
+    config = ConfigParser.RawConfigParser()
+    config.read('/etc/boto.cfg')
```

Comment:
Also fyi the `/etc/boto.cfg` is an old boto2 config file that we didn't remove when upgrading. Since we're not using it for anything else, I think it's fine to still rely on it.

Reply:
Yes, I see. 
As you probably know, in boto2 the proxy configuration was automatically loaded by the file, in boto3 we have to pass it explicitely by a `Config` parameter. 

We could remove it because it isn't required for boto3 but we still need a place from which to load the proxy settings and I agree with you to use this file.